pkgs/development/python-modules/jupyter-contrib-core/default.nix

```
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, jupyter-core
+, lib
+, notebook
```

> [Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.](https://nixos.org/manual/nixpkgs/unstable/#sec-syntax)

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, jupyter-core
, notebook
```

same suggestion for the other files

---

Ok ! I was not aware of this rule.
I fixed it on all 4 files, thanks !